perf/bench/0c34/prep

```diff
@@ -0,0 +1,38 @@
+#! /bin/sh
+#
+# Fetch rules and targets prior for the "r2c" benchmark.
+#
+# rule_dir: input/golang.yaml
+# target_dir: input/govwa
+#
+# Uses sh because bash is not installed within the semgrep docker container.
+#
+set -eu
+
+mkdir -p input
+
+cp golang.yml input/
+cd input
+
+# Obtain a shallow clone of a git repo for a specific commit
+shallow_clone() {
```

Comment:
@mjambon maybe we should move those helpers in a common.sh somewhere and source it from all the prep script?

Reply:
sure, I'll do something like that. It's really just this "shallow_clone" function.